modules/EvseV2G/sdp.cpp

```diff
@@ -252,6 +243,16 @@ int sdp_init(struct v2g_context* v2g_ctx) {
 
     dlog(DLOG_LEVEL_INFO, "SDP socket setup succeeded");
 
+    /* bind only to specified device */
+    if (setsockopt(v2g_ctx->sdp_socket, SOL_SOCKET, SO_BINDTODEVICE, v2g_ctx->if_name, strlen(v2g_ctx->if_name)) ==
+        -1) {
+        dlog(DLOG_LEVEL_ERROR, "setsockopt(SO_BINDTODEVICE) failed: %s", strerror(errno));
+        close(v2g_ctx->sdp_socket);
+        return -1;
+    }
```

Comment:
My intuition says this should go before the `bind()` call, but I couldn't find any clear instructions on that, and didn't test myself.

Reply:
Not sure about that, but it works this way.